src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -504,6 +527,7 @@ def load_arguments(self, command=None):
             command_loaders = self.cmd_to_loader_map.get(command, None)
 
         if command_loaders:
+            command_loaders = self._sort_command_loaders(command_loaders)
```

Comment:
On a second thought, perhaps this line can be moved to L523 so that it only takes effect during ref doc generation?